src/server/editorServices.ts

```diff
@@ -502,7 +522,11 @@ namespace ts.server {
             return scriptInfo && scriptInfo.getDefaultProject();
         }
 
-        private ensureInferredProjectsUpToDate() {
+        /**
+         * Ensures the project structures are upto date

```

Comment:
Typo: "up to"
The documentation for `refreshInferredProjects` is confusing; the name of the method implies that this method ensures that all inferred projects are definitely up to date, then we have a parameter for making really sure?